<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Total Reads and Mapped Reads (TOP2B and P53)</title>

<script src="site_libs/header-attrs-2.29/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.13.2/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-6.5.2/css/all.min.css" rel="stylesheet" />
<link href="site_libs/font-awesome-6.5.2/css/v4-shims.min.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before, .tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "\e259";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "\e258";
  font-family: 'Glyphicons Halflings';
  border: none;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">ChIPSeq_project</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/sayanpaul01/ChIPSeq_project">
    <span class="fa fa-github"></span>
     
     GitHub
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Total Reads and Mapped Reads (TOP2B and
P53)</h1>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2025-08-15
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 6
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 1
</p>
<p>
<strong>Knit directory:</strong> <code>ChIPSeq_project/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.1). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguncommittedchanges">
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> <strong>R Markdown file:</strong> uncommitted
changes </a>
</p>
</div>
<div id="strongRMarkdownfilestronguncommittedchanges"
class="panel-collapse collapse">
<div class="panel-body">
<p>The R Markdown is untracked by Git. To know which version of the R
Markdown file created these results, you‚Äôll want to first commit it to
the Git repo. If you‚Äôre still working on the analysis, you can ignore
this warning. When you‚Äôre finished, you can run
<code>wflow_publish</code> to commit the R Markdown file and build the
HTML.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it‚Äôs best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20250815code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20250815)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20250815code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20250815)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomsayanpaul01ChIPSeqprojecttreed0f4698ff6a03049ec7d39a2c6c3b3912687368etargetblankd0f4698a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/sayanpaul01/ChIPSeq_project/tree/d0f4698ff6a03049ec7d39a2c6c3b3912687368e" target="_blank">d0f4698</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomsayanpaul01ChIPSeqprojecttreed0f4698ff6a03049ec7d39a2c6c3b3912687368etargetblankd0f4698a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/sayanpaul01/ChIPSeq_project/tree/d0f4698ff6a03049ec7d39a2c6c3b3912687368e" target="_blank">d0f4698</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rproj.user/
    Ignored:    analysis/figure/

Untracked files:
    Untracked:  analysis/Total_Reads_and_Mapped_Reads.Rmd
    Untracked:  data/ChIP Seq Summary stat CTCF RAD21.csv
    Untracked:  data/ChIP Seq Summary stat TOP2B P53.csv
    Untracked:  plots_reads_comparison/

</code></pre>
<p>
Note that any generated files, e.g.¬†HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">
<p>
There are no past versions. Publish this analysis with
<code>wflow_publish()</code> to start tracking its development.
</p>
<hr>
</div>
</div>
</div>
<div id="total-reads-by-sample" class="section level2">
<h2><strong>üìå Total Reads by Sample</strong></h2>
<div id="load-required-libraries" class="section level3">
<h3><strong>Load Required Libraries</strong></h3>
<pre class="r"><code># Load necessary R packages
library(edgeR)</code></pre>
<pre><code>Warning: package &#39;edgeR&#39; was built under R version 4.3.2</code></pre>
<pre><code>Loading required package: limma</code></pre>
<pre><code>Warning: package &#39;limma&#39; was built under R version 4.3.1</code></pre>
<pre class="r"><code>library(limma)
library(RColorBrewer)
library(gridExtra)
library(reshape2)
library(data.table)</code></pre>
<pre><code>Warning: package &#39;data.table&#39; was built under R version 4.3.3</code></pre>
<pre><code>
Attaching package: &#39;data.table&#39;</code></pre>
<pre><code>The following objects are masked from &#39;package:reshape2&#39;:

    dcast, melt</code></pre>
<pre class="r"><code>library(tidyverse)</code></pre>
<pre><code>Warning: package &#39;tidyverse&#39; was built under R version 4.3.2</code></pre>
<pre><code>Warning: package &#39;tidyr&#39; was built under R version 4.3.3</code></pre>
<pre><code>Warning: package &#39;readr&#39; was built under R version 4.3.3</code></pre>
<pre><code>Warning: package &#39;purrr&#39; was built under R version 4.3.3</code></pre>
<pre><code>Warning: package &#39;dplyr&#39; was built under R version 4.3.2</code></pre>
<pre><code>Warning: package &#39;stringr&#39; was built under R version 4.3.2</code></pre>
<pre><code>Warning: package &#39;lubridate&#39; was built under R version 4.3.3</code></pre>
<pre><code>‚îÄ‚îÄ Attaching core tidyverse packages ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ tidyverse 2.0.0 ‚îÄ‚îÄ
‚úî dplyr     1.1.4     ‚úî readr     2.1.5
‚úî forcats   1.0.0     ‚úî stringr   1.5.1
‚úî ggplot2   3.5.2     ‚úî tibble    3.2.1
‚úî lubridate 1.9.4     ‚úî tidyr     1.3.1
‚úî purrr     1.0.4     </code></pre>
<pre><code>‚îÄ‚îÄ Conflicts ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ‚îÄ tidyverse_conflicts() ‚îÄ‚îÄ
‚úñ dplyr::between()     masks data.table::between()
‚úñ dplyr::combine()     masks gridExtra::combine()
‚úñ dplyr::filter()      masks stats::filter()
‚úñ dplyr::first()       masks data.table::first()
‚úñ lubridate::hour()    masks data.table::hour()
‚úñ lubridate::isoweek() masks data.table::isoweek()
‚úñ dplyr::lag()         masks stats::lag()
‚úñ dplyr::last()        masks data.table::last()
‚úñ lubridate::mday()    masks data.table::mday()
‚úñ lubridate::minute()  masks data.table::minute()
‚úñ lubridate::month()   masks data.table::month()
‚úñ lubridate::quarter() masks data.table::quarter()
‚úñ lubridate::second()  masks data.table::second()
‚úñ purrr::transpose()   masks data.table::transpose()
‚úñ lubridate::wday()    masks data.table::wday()
‚úñ lubridate::week()    masks data.table::week()
‚úñ lubridate::yday()    masks data.table::yday()
‚úñ lubridate::year()    masks data.table::year()
‚Ñπ Use the conflicted package (&lt;http://conflicted.r-lib.org/&gt;) to force all conflicts to become errors</code></pre>
<pre class="r"><code>library(scales)</code></pre>
<pre><code>Warning: package &#39;scales&#39; was built under R version 4.3.2</code></pre>
<pre><code>
Attaching package: &#39;scales&#39;

The following object is masked from &#39;package:purrr&#39;:

    discard

The following object is masked from &#39;package:readr&#39;:

    col_factor</code></pre>
<pre class="r"><code>library(biomaRt)</code></pre>
<pre><code>Warning: package &#39;biomaRt&#39; was built under R version 4.3.2</code></pre>
<pre class="r"><code>library(cowplot)</code></pre>
<pre><code>Warning: package &#39;cowplot&#39; was built under R version 4.3.2</code></pre>
<pre><code>
Attaching package: &#39;cowplot&#39;

The following object is masked from &#39;package:lubridate&#39;:

    stamp</code></pre>
<pre class="r"><code>library(ggrepel)</code></pre>
<pre><code>Warning: package &#39;ggrepel&#39; was built under R version 4.3.3</code></pre>
<pre class="r"><code>library(corrplot)</code></pre>
<pre><code>Warning: package &#39;corrplot&#39; was built under R version 4.3.3</code></pre>
<pre><code>corrplot 0.95 loaded</code></pre>
<pre class="r"><code>library(Hmisc)</code></pre>
<pre><code>Warning: package &#39;Hmisc&#39; was built under R version 4.3.3</code></pre>
<pre><code>
Attaching package: &#39;Hmisc&#39;

The following objects are masked from &#39;package:dplyr&#39;:

    src, summarize

The following objects are masked from &#39;package:base&#39;:

    format.pval, units</code></pre>
<pre class="r"><code>library(ggpubr)</code></pre>
<pre><code>Warning: package &#39;ggpubr&#39; was built under R version 4.3.1</code></pre>
<pre><code>
Attaching package: &#39;ggpubr&#39;

The following object is masked from &#39;package:cowplot&#39;:

    get_legend</code></pre>
</div>
</div>
<div id="load-data" class="section level2">
<h2><strong>üìç 2. Load Data</strong></h2>
<pre class="r"><code>align&lt;-read.csv(&quot;data/ChIP Seq Summary stat TOP2B P53.csv&quot;)

map&lt;-data.frame(align)

map$Treatment&lt;- factor(map$Treatment, levels = c(&quot;VEH_TOP2B&quot;, &quot;DOX_TOP2B&quot;, &quot;VEH_p53&quot;, &quot;DOX_p53&quot;, &quot;VEH_Input_P53&quot;, &quot;DOX_Input_P53&quot;))</code></pre>
</div>
<div id="define-color-palettes" class="section level2">
<h2><strong>üìç 3. Define Color Palettes</strong></h2>
<pre class="r"><code>drug_palc &lt;- c(&quot;#8B006D&quot;,&quot;#DF707E&quot;,&quot;#F1B72B&quot;, &quot;#3386DD&quot;,&quot;#707031&quot;,&quot;#41B333&quot;)

Ind_palc &lt;- c(&quot;#ffbe0b&quot;,&quot;#ff006e&quot;,&quot;#fb5607&quot;, &quot;#8338ec&quot;,&quot;#3a86ff&quot;,&quot;#4a4e69&quot;)

Treat_palc &lt;- c(&quot;#ffbe0b&quot;,&quot;#ff006e&quot;,&quot;#fb5607&quot;, &quot;#8338ec&quot;, &quot;#800080&quot;,&quot;#FFC0CB&quot;)

Map_palc &lt;- c(&quot;#9b19f5&quot;,&quot;#e6d800&quot;, &quot;#b3d4ff&quot;)

Combined_palc &lt;- c(&quot;#FF0000&quot;,&quot;#00FF00&quot;,&quot;#0000FF&quot;,&quot;#FFFF00&quot;,&quot;#FF00FF&quot;,&quot;#00FFFF&quot;,&quot;#FFA500&quot;,&quot;#800080&quot;,&quot;#FFC0CB&quot;,&quot;#A52A2A&quot;,&quot;#808080&quot;,&quot;#FFD700&quot;)

Type_palc &lt;- c(&quot;#800080&quot;,&quot;#FFD700&quot;)</code></pre>
</div>
<div id="prepare-data" class="section level2">
<h2><strong>üìç 4. Prepare Data</strong></h2>
<pre class="r"><code># Factor Sample_name to maintain order
map$Sample.Det&lt;-factor(map$Sample.Det,levels = map$Sample.Det)</code></pre>
</div>
<div id="plot-total-reads-by-sample" class="section level2">
<h2><strong>üìç 5. Plot Total Reads by Sample</strong></h2>
<pre class="r"><code>map %&gt;%
  #mutate(Drug=factor(Drug,levels=c(&quot;CX-5461&quot;,&quot;DOX&quot;,&quot;VEH&quot;))) %&gt;%
  #mutate(Conc.=factor(Conc.,levels=c(&quot;0.1&quot;,&quot;0.5&quot;))) %&gt;%
  #mutate(Time=factor(Time,levels=c(&quot;3&quot;,&quot;24&quot;,&quot;48&quot;))) %&gt;%
  #group_by(Drug,Conc.,Time) %&gt;%
  ggplot(., aes (x =Sample.Det, y=Total.Reads..before.trimming., fill = Ind))+
  geom_col()+
  #geom_hline(aes(yintercept=20000000))+
  scale_fill_manual(values=Ind_palc)+
  scale_y_continuous(labels = label_number(suffix = &quot; M&quot;, scale = 1e-6))+
  ggtitle(expression(&quot;Total number of reads by sample&quot;))+
  xlab(&quot;&quot;)+
  ylab(expression(&quot;ChIP -sequencing reads&quot;))+
  theme_bw()+
  theme(plot.title = element_text(size = rel(2), hjust = 0.5),
        axis.title = element_text(size = 15, color = &quot;black&quot;),
        axis.ticks = element_line(linewidth = 1.5),
        axis.line = element_line(linewidth = 1.5),
        axis.text.y = element_text(size =10, color = &quot;black&quot;, angle = 0, hjust = 0.8, vjust = 0.5),
        axis.text.x = element_text(size =10, color = &quot;black&quot;, angle = 90, hjust = 1, vjust = 0.2),
        #strip.text.x = element_text(size = 15, color = &quot;black&quot;, face = &quot;bold&quot;),
        strip.text.y = element_text(color = &quot;white&quot;))</code></pre>
<p><img src="figure/Total_Reads_and_Mapped_Reads.Rmd/plot-total-reads-1.png" width="1728" style="display: block; margin: auto;" /></p>
</div>
<div id="total-reads-by-individuals" class="section level2">
<h2><strong>üìå Total Reads by Individuals</strong></h2>
<pre class="r"><code>map %&gt;%
  ggplot(aes(x = Ind, y = Total.Reads..before.trimming., fill = Ind)) +
  geom_boxplot() +
  scale_fill_manual(values = Ind_palc) +
  scale_y_continuous(labels = label_number(suffix = &quot; M&quot;, scale = 1e-6),
                     limits = c(0, NA)) +
  ggtitle(expression(&quot;Total number of reads by Individual&quot;)) +
  xlab(&quot;&quot;) +
  ylab(expression(&quot;ChIP -sequencing reads&quot;)) +
  theme_bw() +
  theme(
    plot.title  = element_text(size = rel(2), hjust = 0.5),
    axis.title  = element_text(size = 15, color = &quot;black&quot;),
    axis.ticks  = element_line(linewidth = 1.5),
    axis.line   = element_line(linewidth = 1.5),
    axis.text.y = element_text(size = 10, color = &quot;black&quot;),
    axis.text.x = element_text(size = 10, color = &quot;black&quot;, angle = 90,
                               hjust = 1, vjust = 0.2),
    strip.text.y = element_text(color = &quot;white&quot;)
  )</code></pre>
<p><img src="figure/Total_Reads_and_Mapped_Reads.Rmd/generate_boxplot%20indv-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="total-reads-by-treatment" class="section level2">
<h2><strong>üìå Total Reads by Treatment</strong></h2>
<pre class="r"><code>map %&gt;%
  ggplot(aes(x = Treatment, y = Total.Reads..before.trimming., fill = Treatment)) +
  geom_boxplot() +
  scale_fill_manual(values = Treat_palc) +
  scale_y_continuous(labels = label_number(suffix = &quot; M&quot;, scale = 1e-6),
                     limits = c(0, NA)) +
  ggtitle(expression(&quot;Total number of reads by Treatment&quot;)) +
  xlab(&quot;&quot;) +
  ylab(expression(&quot;ChIP -sequencing reads&quot;)) +
  theme_bw() +
  theme(
    plot.title  = element_text(size = rel(2), hjust = 0.5),
    axis.title  = element_text(size = 15, color = &quot;black&quot;),
    axis.ticks  = element_line(linewidth = 1.5),
    axis.line   = element_line(linewidth = 1.5),
    axis.text.y = element_text(size = 10, color = &quot;black&quot;),
    axis.text.x = element_text(size = 10, color = &quot;black&quot;, angle = 90,
                               hjust = 1, vjust = 0.2),
    strip.text.y = element_text(color = &quot;white&quot;)
  )</code></pre>
<p><img src="figure/Total_Reads_and_Mapped_Reads.Rmd/generate_boxplot%20treatment-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="total-reads-by-sample-type" class="section level2">
<h2><strong>üìå Total Reads by Sample type</strong></h2>
<pre class="r"><code>map %&gt;%
  ggplot(aes(x = Type, y = Total.Reads..before.trimming., fill = Type)) +
  geom_boxplot() +
  scale_fill_manual(values = Type_palc) +
  scale_y_continuous(labels = label_number(suffix = &quot; M&quot;, scale = 1e-6),
                     limits = c(0, NA)) +
  ggtitle(expression(&quot;Total number of reads by Sample Type&quot;)) +
  xlab(&quot;&quot;) +
  ylab(expression(&quot;ChIP -sequencing reads&quot;)) +
  theme_bw() +
  theme(
    plot.title  = element_text(size = rel(2), hjust = 0.5),
    axis.title  = element_text(size = 15, color = &quot;black&quot;),
    axis.ticks  = element_line(linewidth = 1.5),
    axis.line   = element_line(linewidth = 1.5),
    axis.text.y = element_text(size = 10, color = &quot;black&quot;),
    axis.text.x = element_text(size = 10, color = &quot;black&quot;, angle = 90,
                               hjust = 1, vjust = 0.2),
    strip.text.y = element_text(color = &quot;white&quot;)
  )</code></pre>
<p><img src="figure/Total_Reads_and_Mapped_Reads.Rmd/generate_boxplot%20type-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="total-reads-by-individuals-and-sample-type"
class="section level2">
<h2><strong>üìå Total Reads by Individuals and Sample type</strong></h2>
<pre class="r"><code>map %&gt;%
  ggplot(aes(x = Ind, y = Total.Reads..before.trimming., fill = Type)) +
  geom_boxplot() +
  scale_fill_manual(values = Type_palc) +
  scale_y_continuous(labels = label_number(suffix = &quot; M&quot;, scale = 1e-6),
                     limits = c(0, NA)) +
  ggtitle(expression(&quot;Total number of reads by Individual and Sample Type&quot;)) +
  xlab(&quot;&quot;) +
  ylab(expression(&quot;ChIP -sequencing reads&quot;)) +
  theme_bw() +
  theme(
    plot.title  = element_text(size = rel(2), hjust = 0.5),
    axis.title  = element_text(size = 15, color = &quot;black&quot;),
    axis.ticks  = element_line(linewidth = 1.5),
    axis.line   = element_line(linewidth = 1.5),
    axis.text.y = element_text(size = 10, color = &quot;black&quot;),
    axis.text.x = element_text(size = 10, color = &quot;black&quot;, angle = 90,
                               hjust = 1, vjust = 0.2),
    strip.text.y = element_text(color = &quot;white&quot;)
  )</code></pre>
<p><img src="figure/Total_Reads_and_Mapped_Reads.Rmd/generate_boxplot%20Ind%20and%20type-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="number-of-reads-before-and-after-trimming"
class="section level2">
<h2><strong>üìå Number of reads before and after trimming</strong></h2>
<pre class="r"><code>library(dplyr)
library(tidyr)
library(ggplot2)
library(scales)

# Keep sample order as in file
map$Sample.Det &lt;- factor(map$Sample.Det, levels = map$Sample.Det)

# Make sure read counts are numeric (in case CSV parsed as character)
map &lt;- map %&gt;%
  mutate(
    `Total.Reads..before.trimming.` = as.numeric(`Total.Reads..before.trimming.`),
    `Total.reads..after.Trimming.`  = as.numeric(`Total.reads..after.Trimming.`)
  )

# Compute kept fraction for annotation
map &lt;- map %&gt;%
  mutate(kept_frac = `Total.reads..after.Trimming.` / `Total.Reads..before.trimming.`)

# Long format for before vs after
map_long &lt;- map %&gt;%
  pivot_longer(
    cols = c(`Total.Reads..before.trimming.`, `Total.reads..after.Trimming.`),
    names_to = &quot;TrimStage&quot;, values_to = &quot;Reads&quot;
  ) %&gt;%
  mutate(
    TrimStage = factor(
      TrimStage,
      levels = c(&quot;Total.Reads..before.trimming.&quot;, &quot;Total.reads..after.Trimming.&quot;),
      labels = c(&quot;Before trimming&quot;, &quot;After trimming&quot;)
    )
  )

# High-contrast (colorblind-safe) colors
stage_colors &lt;- c(&quot;Before trimming&quot; = &quot;#1f78b4&quot;,  # deep blue
                  &quot;After trimming&quot;  = &quot;#ff7f00&quot;)  # bright orange

# Base plot
p &lt;- ggplot(map_long, aes(x = Sample.Det, y = Reads, fill = TrimStage)) +
  geom_col(position = position_dodge(width = 0.8), width = 0.75, color = &quot;black&quot;) +
  scale_fill_manual(values = stage_colors) +
  scale_y_continuous(labels = label_number(suffix = &quot; M&quot;, scale = 1e-6)) +
  ggtitle(expression(&quot;Total number of reads by sample: before vs after trimming&quot;)) +
  xlab(&quot;&quot;) +
  ylab(expression(&quot;ChIP-sequencing reads&quot;)) +
  theme_bw() +
  theme(
    plot.title  = element_text(size = rel(2), hjust = 0.5),
    axis.title  = element_text(size = 15, color = &quot;black&quot;),
    axis.ticks  = element_line(linewidth = 1.5),
    axis.line   = element_line(linewidth = 1.5),
    axis.text.y = element_text(size = 10, color = &quot;black&quot;),
    axis.text.x = element_text(size = 10, color = &quot;black&quot;, angle = 90, hjust = 1, vjust = 0.2),
    legend.title = element_blank()
  )
p</code></pre>
<p><img src="figure/Total_Reads_and_Mapped_Reads.Rmd/plot%20trimmed%20reads-1.png" width="1728" style="display: block; margin: auto;" /></p>
</div>
<div
id="mean-sequencing-reads-before-and-after-trimming-by-individual-sample-type-and-antibody"
class="section level2">
<h2><strong>üìå Mean Sequencing Reads Before and After Trimming by
Individual, Sample Type, and Antibody</strong></h2>
<pre class="r"><code>library(dplyr)
library(tidyr)
library(ggplot2)
library(scales)
library(stringr)

# --- Prep -------------------------------------------------------------

# Use your existing data frame name: map (change to df if needed)
df &lt;- map

# Ensure required columns exist / are numeric
df &lt;- df %&gt;%
  mutate(
    `Total.Reads..before.trimming.` = as.numeric(`Total.Reads..before.trimming.`),
    `Total.reads..after.Trimming.`  = as.numeric(`Total.reads..after.Trimming.`)
  )

# Define Tx if your data uses &#39;Treatment&#39;
if (!&quot;Tx&quot; %in% names(df) &amp;&amp; &quot;Treatment&quot; %in% names(df)) {
  df &lt;- df %&gt;% mutate(Tx = Treatment)
}

# Normalize factors you care about
df &lt;- df %&gt;%
  mutate(
    Ab   = ifelse(is.na(Ab) | Ab == &quot;&quot;, &quot;Input&quot;, as.character(Ab)),
    Type = factor(Type, levels = c(&quot;ChIP_DNA&quot;, &quot;Input_DNA&quot;))  # adjust if your labels differ
  )

# Helper: label function for Y axis (millions)
reads_lab &lt;- label_number(suffix = &quot; M&quot;, scale = 1e-6)

# Kept fraction (After / Before)
df &lt;- df %&gt;%
  mutate(kept_frac = `Total.reads..after.Trimming.` / `Total.Reads..before.trimming.`)

# --- Group, summarise, pivot -----------------------------------------

group_sum &lt;- df %&gt;%
  group_by(Ind, Type, Ab, Tx) %&gt;%
  summarise(
    before_mean = mean(`Total.Reads..before.trimming.`, na.rm = TRUE),
    after_mean  = mean(`Total.reads..after.Trimming.`,  na.rm = TRUE),
    kept_pct    = 100 * mean(kept_frac, na.rm = TRUE),
    .groups     = &quot;drop&quot;
  ) %&gt;%
  pivot_longer(
    cols      = c(before_mean, after_mean),
    names_to  = &quot;stage&quot;,
    values_to = &quot;reads&quot;
  ) %&gt;%
  mutate(
    stage = factor(stage, levels = c(&quot;before_mean&quot;, &quot;after_mean&quot;),
                          labels = c(&quot;Before&quot;, &quot;After&quot;)),
    facet_id = paste0(Ind, &quot;_&quot;, Type, &quot;_&quot;, Ab)
  )

# Order facets by Ind then Type (cleaner viewing)
facet_levels &lt;- group_sum %&gt;%
  distinct(Ind, Type, Ab, facet_id) %&gt;%
  arrange(Ind, Type, Ab) %&gt;%
  pull(facet_id)

group_sum &lt;- group_sum %&gt;%
  mutate(facet_id = factor(facet_id, levels = facet_levels))

# Optional: prettier strip labels (multi-line)
strip_labeller &lt;- function(ids) {
  parts &lt;- str_split(ids, &quot;_&quot;, n = 3, simplify = TRUE)
  paste0(&quot;Ind: &quot;, parts[,1], &quot;\nType: &quot;, parts[,2], &quot;\nAb: &quot;, parts[,3])
}

# Colors for Before / After
before_after_colors &lt;- c(&quot;Before&quot; = &quot;#1f78b4&quot;, &quot;After&quot; = &quot;#ff7f00&quot;)

# --- Plot -------------------------------------------------------------

p_group &lt;- ggplot(group_sum, aes(x = Tx, y = reads, fill = stage)) +
  geom_col(position = position_dodge(width = 0.8), width = 0.7, color = &quot;black&quot;) +
  scale_fill_manual(values = before_after_colors) +
  facet_wrap(~ facet_id, scales = &quot;free_y&quot;,
             labeller = labeller(facet_id = strip_labeller)) +
  scale_y_continuous(labels = reads_lab) +
  labs(
    x = &quot;Treatment&quot;,
    y = &quot;Mean reads&quot;,
    title = &quot;Mean reads by Individual √ó Sample Type √ó Antibody&quot;
  ) +
  theme_bw(base_size = 12) +
  theme(
    legend.title = element_blank(),
    plot.title = element_text(hjust = 0.5, face = &quot;bold&quot;),
    strip.text  = element_text(size = 10)
  )

p_group</code></pre>
<p><img src="figure/Total_Reads_and_Mapped_Reads.Rmd/plot%20mean%20reads-1.png" width="1728" style="display: block; margin: auto;" /></p>
</div>
<div id="duplication-percentage-by-sample" class="section level2">
<h2><strong>üìå Duplication percentage by Sample</strong></h2>
<pre class="r"><code>map %&gt;%
  ggplot(aes(x = Sample.Det, y = Duplication.percentage, fill = Ind)) +
  geom_col(color = &quot;black&quot;) +
  scale_fill_manual(values = Ind_palc) +
  scale_y_continuous(limits = c(0, 100), expand = c(0, 0)) +
  ggtitle(expression(&quot;Duplication Percentage by Sample&quot;)) +
  xlab(&quot;&quot;) +
  ylab(expression(&quot;Duplication percentage&quot;)) +
  theme_bw() +
  theme(
    plot.title  = element_text(size = rel(2), hjust = 0.5),
    axis.title  = element_text(size = 15, color = &quot;black&quot;),
    axis.ticks  = element_line(linewidth = 1.5),
    axis.line   = element_line(linewidth = 1.5),
    axis.text.y = element_text(size = 10, color = &quot;black&quot;),
    axis.text.x = element_text(size = 10, color = &quot;black&quot;, angle = 90,
                               hjust = 1, vjust = 0.2),
    strip.text.y = element_text(color = &quot;white&quot;)
  )</code></pre>
<p><img src="figure/Total_Reads_and_Mapped_Reads.Rmd/Dup-1.png" width="1728" style="display: block; margin: auto;" /></p>
</div>
<div id="duplication-percentage-by-individual" class="section level2">
<h2><strong>üìå Duplication percentage by Individual</strong></h2>
<pre class="r"><code>map %&gt;%
  ggplot(aes(x = Ind, y = Duplication.percentage, fill = Ind)) +
  geom_boxplot() +
  scale_fill_manual(values = Ind_palc) +
  scale_y_continuous(limits = c(0, 100), labels = function(x) paste0(x, &quot;%&quot;)) +
  ggtitle(expression(&quot;Duplication percentage by Individual&quot;)) +
  xlab(&quot;&quot;) +
  ylab(expression(&quot;Duplication percentage&quot;)) +
  theme_bw() +
  theme(
    plot.title  = element_text(size = rel(2), hjust = 0.5),
    axis.title  = element_text(size = 15, color = &quot;black&quot;),
    axis.ticks  = element_line(linewidth = 1.5),
    axis.line   = element_line(linewidth = 1.5),
    axis.text.y = element_text(size = 10, color = &quot;black&quot;),
    axis.text.x = element_text(size = 10, color = &quot;black&quot;, angle = 90,
                               hjust = 1, vjust = 0.2),
    strip.text.y = element_text(color = &quot;white&quot;)
  )</code></pre>
<p><img src="figure/Total_Reads_and_Mapped_Reads.Rmd/Dup1-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="duplication-percentage-by-treatment" class="section level2">
<h2><strong>üìå Duplication percentage by Treatment</strong></h2>
<pre class="r"><code>map %&gt;%
  ggplot(aes(x = Treatment, y = Duplication.percentage, fill = Treatment)) +
  geom_boxplot() +
  scale_fill_manual(values = Treat_palc) +
  scale_y_continuous(limits = c(0, 100), labels = function(x) paste0(x, &quot;%&quot;)) +
  ggtitle(expression(&quot;Duplication percentage by Treatment&quot;)) +
  xlab(&quot;&quot;) +
  ylab(expression(&quot;Duplication percentage&quot;)) +
  theme_bw() +
  theme(
    plot.title  = element_text(size = rel(2), hjust = 0.5),
    axis.title  = element_text(size = 15, color = &quot;black&quot;),
    axis.ticks  = element_line(linewidth = 1.5),
    axis.line   = element_line(linewidth = 1.5),
    axis.text.y = element_text(size = 10, color = &quot;black&quot;),
    axis.text.x = element_text(size = 10, color = &quot;black&quot;, angle = 90,
                               hjust = 1, vjust = 0.2),
    strip.text.y = element_text(color = &quot;white&quot;)
  )</code></pre>
<p><img src="figure/Total_Reads_and_Mapped_Reads.Rmd/Dup2-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="duplication-percentage-by-sample-type" class="section level2">
<h2><strong>üìå Duplication percentage by Sample type</strong></h2>
<pre class="r"><code>map %&gt;%
  ggplot(aes(x = Type, y = Duplication.percentage, fill = Type)) +
  geom_boxplot() +
  scale_fill_manual(values = Type_palc) +
  scale_y_continuous(limits = c(0, 100), labels = function(x) paste0(x, &quot;%&quot;)) +
  ggtitle(expression(&quot;Duplication percentage by Sample Type&quot;)) +
  xlab(&quot;&quot;) +
  ylab(expression(&quot;Duplication percentage&quot;)) +
  theme_bw() +
  theme(
    plot.title  = element_text(size = rel(2), hjust = 0.5),
    axis.title  = element_text(size = 15, color = &quot;black&quot;),
    axis.ticks  = element_line(linewidth = 1.5),
    axis.line   = element_line(linewidth = 1.5),
    axis.text.y = element_text(size = 10, color = &quot;black&quot;),
    axis.text.x = element_text(size = 10, color = &quot;black&quot;, angle = 90,
                               hjust = 1, vjust = 0.2),
    strip.text.y = element_text(color = &quot;white&quot;)
  )</code></pre>
<p><img src="figure/Total_Reads_and_Mapped_Reads.Rmd/Dup3-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="duplication-percentage-by-individual-and-sample-type"
class="section level2">
<h2><strong>üìå Duplication percentage by Individual and Sample
type</strong></h2>
<pre class="r"><code>map %&gt;%
  ggplot(aes(x = Ind, y = Duplication.percentage, fill = Type)) +
  geom_boxplot() +
  scale_fill_manual(values = Type_palc) +
  scale_y_continuous(limits = c(0, 100), labels = function(x) paste0(x, &quot;%&quot;)) +
  ggtitle(expression(&quot;Duplication percentage by Individual and Sample Type&quot;)) +
  xlab(&quot;&quot;) +
  ylab(expression(&quot;Duplication percentage&quot;)) +
  theme_bw() +
  theme(
    plot.title  = element_text(size = rel(2), hjust = 0.5),
    axis.title  = element_text(size = 15, color = &quot;black&quot;),
    axis.ticks  = element_line(linewidth = 1.5),
    axis.line   = element_line(linewidth = 1.5),
    axis.text.y = element_text(size = 10, color = &quot;black&quot;),
    axis.text.x = element_text(size = 10, color = &quot;black&quot;, angle = 90,
                               hjust = 1, vjust = 0.2),
    strip.text.y = element_text(color = &quot;white&quot;)
  )</code></pre>
<p><img src="figure/Total_Reads_and_Mapped_Reads.Rmd/Dup4-1.png" width="672" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.3.0 (2023-04-21 ucrt)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 11 x64 (build 26100)

Matrix products: default


locale:
[1] LC_COLLATE=English_United States.utf8 
[2] LC_CTYPE=English_United States.utf8   
[3] LC_MONETARY=English_United States.utf8
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.utf8    

time zone: America/Chicago
tzcode source: internal

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] ggpubr_0.6.0       Hmisc_5.2-3        corrplot_0.95      ggrepel_0.9.6     
 [5] cowplot_1.1.3      biomaRt_2.58.2     scales_1.3.0       lubridate_1.9.4   
 [9] forcats_1.0.0      stringr_1.5.1      dplyr_1.1.4        purrr_1.0.4       
[13] readr_2.1.5        tidyr_1.3.1        tibble_3.2.1       ggplot2_3.5.2     
[17] tidyverse_2.0.0    data.table_1.17.0  reshape2_1.4.4     gridExtra_2.3     
[21] RColorBrewer_1.1-3 edgeR_4.0.16       limma_3.58.1      

loaded via a namespace (and not attached):
 [1] DBI_1.2.3               bitops_1.0-9            rlang_1.1.3            
 [4] magrittr_2.0.3          git2r_0.36.2            compiler_4.3.0         
 [7] RSQLite_2.3.9           png_0.1-8               vctrs_0.6.5            
[10] pkgconfig_2.0.3         crayon_1.5.3            fastmap_1.2.0          
[13] backports_1.5.0         dbplyr_2.5.0            XVector_0.42.0         
[16] labeling_0.4.3          promises_1.3.2          rmarkdown_2.29         
[19] tzdb_0.5.0              bit_4.6.0               xfun_0.52              
[22] zlibbioc_1.48.2         cachem_1.1.0            GenomeInfoDb_1.38.8    
[25] jsonlite_2.0.0          progress_1.2.3          blob_1.2.4             
[28] later_1.3.2             broom_1.0.8             prettyunits_1.2.0      
[31] cluster_2.1.8.1         R6_2.6.1                bslib_0.9.0            
[34] stringi_1.8.3           car_3.1-3               rpart_4.1.24           
[37] jquerylib_0.1.4         Rcpp_1.0.12             knitr_1.50             
[40] base64enc_0.1-3         IRanges_2.36.0          httpuv_1.6.15          
[43] nnet_7.3-20             timechange_0.3.0        tidyselect_1.2.1       
[46] abind_1.4-8             rstudioapi_0.17.1       yaml_2.3.10            
[49] curl_6.2.2              lattice_0.22-7          plyr_1.8.9             
[52] Biobase_2.62.0          withr_3.0.2             KEGGREST_1.42.0        
[55] evaluate_1.0.3          foreign_0.8-90          BiocFileCache_2.10.2   
[58] xml2_1.3.8              Biostrings_2.70.3       pillar_1.10.2          
[61] filelock_1.0.3          carData_3.0-5           checkmate_2.3.2        
[64] stats4_4.3.0            generics_0.1.3          rprojroot_2.0.4        
[67] RCurl_1.98-1.17         S4Vectors_0.40.2        hms_1.1.3              
[70] munsell_0.5.1           glue_1.7.0              tools_4.3.0            
[73] ggsignif_0.6.4          locfit_1.5-9.12         fs_1.6.3               
[76] XML_3.99-0.18           grid_4.3.0              AnnotationDbi_1.64.1   
[79] colorspace_2.1-0        GenomeInfoDbData_1.2.11 htmlTable_2.4.3        
[82] Formula_1.2-5           cli_3.6.1               rappdirs_0.3.3         
[85] workflowr_1.7.1         gtable_0.3.6            rstatix_0.7.2          
[88] sass_0.4.10             digest_0.6.34           BiocGenerics_0.48.1    
[91] farver_2.1.2            htmlwidgets_1.6.4       memoise_2.0.1          
[94] htmltools_0.5.8.1       lifecycle_1.0.4         httr_1.4.7             
[97] statmod_1.5.0           bit64_4.6.0-1          </code></pre>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
